pkg/sql/logictest/testdata/logic_test/row_level_ttl

```diff
@@ -1199,3 +1217,75 @@ statement ok
 DROP TABLE "Table-Name"
 
 subtest end
+
+subtest set_both_reset_ttl_expire_after
```

Comment:
nit: can we renmae "set_both_ZZ" to "set_both_xx_and_yy_ZZ" so we know both of what and what are set. Or maybe put this in a comment

Reply:
Done in a commit to make it more readable.